app/js/views/reminders.jsx

```diff
@@ -101,6 +105,70 @@ export default class Reminders extends React.Component {
       });
   }
 
+  /**
+   * Generate a phrase to be spoken to confirm a reminder.
+   *
+   * @param {Object} reminder
+   * @returns {string}
+   */
+  confirmAReminder(reminder) {
+    const users = reminder.recipients
+      .map((user) => user === 'me' ? 'you' : user)
+      .join(' and ');
```

Comment:
Yes I know. I plan to use `cldr-js` in the future. I can actually work on this as a followup to that PR.